bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -615,7 +616,7 @@ Dialog_GotoMapLocation_Label_LocationLatitude                          = L&atitu
 Dialog_GotoMapLocation_Label_LocationLongitude                         = L&ongitude
 Dialog_GotoMapLocation_Message_LatLonIsPasted_FromClipboard            = Latitude and longitude was filled from the clipboard:\n\
                                                                          {0}
-Dialog_GotoMapLocation_Message_LatLonIsPasted_FromMouseMapPosition     = Latitude and longitude was filled from the current mouse map position 
+Dialog_GotoMapLocation_Message_LatLonIsPasted_FromMouseMapPosition     = Latitude and longitude were filled from the current mouse map position 
```

Comment:
My first version was "Latitude and longitude is filled ..."

I'm always struggeling with some words :-)

Reply:
That's why it's good to have several "pair of eyes" looking at the code/translation 👍